README.md

```diff
@@ -8,58 +8,66 @@ are showed accordingly with the response of the previous ones.
 Calling the component:
 
 ### questions.js
+
 ```javascript
 export default = [
   questionNumber: 1,
   inputType: 'twoOptionsButtons',
   inputClassNames: 'optionsCssClass',
   questionText: 'Who is your favorite hero?',
-  optionOneText: 'Spider-Man',
-  optionTwoText: 'Batman',
-  descriptionTextClassNames: 'descroptionCssClass',
-  descriptionText: (
-    <span>
+  questionTypeFields: {
```

Comment:
Thinking about the structure of this problem seems that we are creating a kind of tree, so we could try to be a bit more generic and avoid "fixed names", like  `optionOneText` and `optionTwoText`.

I'm not sure if it could work in this scenario - or the implications of this strategy, but if we have something like that:

```javascript
question: {
    type: 'twoOptionsButtons'
    questionText: 'Who is your favorite hero?',
    ...
    answers: [{
        text: 'Spider-Man'
        question: {...}
    },
    {
       text: 'Batman'
       question: {...}
     }],
}
```

Reply:
Nice one! This will let us to expand more easily the options for question in the future